src/commands/utils/config.ts

```diff
@@ -38,93 +39,99 @@ export function getKustomizeConfig(): Errorable<KustomizeConfig> {
+    const config = getConfigValue(kubectlGadgetConfig, "releaseTag");
 
     if (failed(config)) {
         return {
             succeeded: false,
-            error: `Failed to read azure.kubectlgadget configuration: ${config.error}`
+            error: `Failed to read azure.kubectlgadget configuration: ${config.error}`,
         };
     }
 
     const configresult = {
-        releaseTag: config.result
+        releaseTag: config.result,
     };
 
     return { succeeded: true, result: configresult };
 }
 
 export function getDraftConfig(): Errorable<DraftConfig> {
-    const draftConfig = vscode.workspace.getConfiguration('aks.drafttool');
-    const props = getConfigValue(draftConfig, 'releaseTag');
- 
+    const draftConfig = vscode.workspace.getConfiguration("aks.drafttool");
+    const props = getConfigValue(draftConfig, "releaseTag");
+
     if (failed(props)) {
-       return {
-          succeeded: false,
-          error: `Failed to read aks.draft configuration: ${props.error}`
-       };
+        return {
+            succeeded: false,
+            error: `Failed to read aks.draft configuration: ${props.error}`,
+        };
     }
- 
+
     const config = {
-       releaseTag: props.result
+        releaseTag: props.result,
     };
 
-    return {succeeded: true, result: config};
- }
+    return { succeeded: true, result: config };
+}
 
 function getConfigValue(config: vscode.WorkspaceConfiguration, key: string): Errorable<string> {
     const value = config.get(key);
     if (value === undefined) {
-        return { succeeded: false, error: `${key} not defined.` }
+        return { succeeded: false, error: `${key} not defined.` };
     }
     const result = value as string;
     if (result === undefined) {
-        return { succeeded: false, error: `${key} value has type: ${typeof value}; expected string.` }
+        return { succeeded: false, error: `${key} value has type: ${typeof value}; expected string.` };
     }
     return { succeeded: true, result: result };
 }
 
 export function getKubectlCustomCommands(): PresetCommand[] {
-    const config = vscode.workspace.getConfiguration('azure.customkubectl');
-    const value = config.get('commands');
+    const config = vscode.workspace.getConfiguration("azure.customkubectl");
+    const value = config.get("commands");
     if (!Array.isArray(value)) {
         return [];
     }
 
-    return value.filter(isCommand).map(item => ({...item, category: CommandCategory.Custom}));
+    return value.filter(isCommand).map((item) => ({ ...item, category: CommandCategory.Custom }));
 
-    function isCommand(value: any): value is PresetCommand {
-        return (value.constructor.name === 'Object') && (value as PresetCommand).command && (value as PresetCommand).name ? true : false;
+    function isCommand(value: unknown): value is PresetCommand {
+        return isObject(value) && "command" in value && "name" in value;
```

Comment:
This should be equivalent to the previous code, except the old code wouldn't handle nulls, whereas `isObject` does.